atd-vze/src/views/Crashes/RelatedRecordsTable.js

```diff
@@ -271,7 +268,8 @@ const RelatedRecordsTable = ({
                             )}
 
                             {!isEditing &&
-                              (fieldConfig.fields[field].badge ? (
+                              (fieldConfig.fields[field].badge &&
+                              formatValue(row, field) != "NO DATA" ? (
```

Comment:
for injury severity we dont want a badge if its null, we want the styled NO DATA value to match the rest of the table

Reply:
oo good catch thanks mike! :monocle_face: